docs/notification-scripts.md

```diff
@@ -25,6 +28,15 @@ The interface for notification scripts is as follows:
   * The message body, sanitized of tabs (converted to spaces) and escape
     characters (converted to "<ESC>").
 
+* Version 2 and later: Matterhorn will invoke the command with zero
```

Comment:
This should probably say "Version 2" (not "and later") since "2" is the only other supported version at this point.

Reply:
Resolved by commit 07fb71e2 .

Reply:
Resolved by commit 07fb71e .